src/components/SideMenu/SideMenu.component.jsx

```diff
@@ -1,19 +1,22 @@
 import React from 'react';
+import { useHistory } from 'react-router-dom';
 import HomeOutlinedIcon from '@material-ui/icons/HomeOutlined';
 import ThumbUpAltOutlinedIcon from '@material-ui/icons/ThumbUpAltOutlined';
 import ExitToAppOutlinedIcon from '@material-ui/icons/ExitToAppOutlined';
 
 import { SideMenuNav, SideMenuItems, SideMenuLink } from './SideMenu.styled';
 
 function SideMenu({ sideMenuAction, handleToggleMenu }) {
+  const { push } = useHistory();
+
   return (
     <SideMenuNav
       className={sideMenuAction ? 'open-menu' : ''}
       onClick={() => handleToggleMenu(false)}
       tabIndex="0"
     >
       <ul aria-label="navbar-list">
-        <SideMenuItems className="centered-items" tabIndex="0">
+        <SideMenuItems className="centered-items" tabIndex="0" onClick={() => push('/')}>
```

Comment:
por qué no usar un `Link`?

Reply:
Es que lo usé en el CardComponent y cuando corrí el test me salió  `Invariant failed: You should not use <Link> outside a <Router>` por eso busqué una alternativa y encontré el "push". Ya lo cambié a Link y me salé el mismo mensaje, hay alguna forma para que esto no salte en los test? :s